doc/application_workflow/voting_app.md

```diff
@@ -0,0 +1,106 @@
+# Voting App on VIC
+
+The [voting app](https://github.com/docker/example-voting-app) is one of Docker's example multi-tiered app.  We will neither discuss the design of the app nor explain how it works.  Instead, we will focus on it's Docker Compose yml file and use the [guidelines](README.md) to modify that yml file to make it work on VIC 0.7.0.  You can find the modifed compose yml file in the folder [vic/demos/compose/voting-app/](../../demos/compose/voting-app).  We have only included the modifed yml file as the below workflow will use the original source from github.
+
+## Workflow
+
+### Original Compose File
+
+```
+version: "2"
+
+services:
+  vote:
+    build: ./vote
+    command: python app.py
+    volumes:
+     - ./vote:/app
+    ports:
+      - "5000:80"
+
+  redis:
+    image: redis:alpine
+    ports: ["6379"]
+
+  worker:
+    build: ./worker
+
+  db:
+    image: postgres:9.4
+
+  result:
+    build: ./result
+    command: nodemon --debug server.js
+    volumes:
+      - ./result:/app
+    ports:
+      - "5001:80"
+      - "5858:5858"
+```
+
+We see the above compose file are using two features that are not yet supported in VIC 0.7.0.  The first is docker build.  The second is local folder mapping to container volume.  Let's walk through modifying this app and deploy it onto a vSphere environment.
+
+### Getting the App Prepared
+
+First, clone the repository from github.  Note, we have included the modified compose file in our /demos folder, but in this exercise, we are going to modify this app from the sources from github.
+
+Second, to get around the docker build directive, we follow the previously mentioned guidelines and use regular docker to build each component that requires a build.  Then we tag the the images to upload to our private registry (or private account on Docker Hub).  In this example, we are going to use VMWare's victest account on Docker Hub.  You will not be able to use this account, but you can create your own and use that in place of the victest keywords below.  Please note, the steps shown below are performed in a terminal using regular docker (as opposed to VIC's docker personality daemon).
+
+**build the images:**  
```

Comment:
No, but I can see how it maybe useful for readers that already have access to container images but not the original Dockerfile.